src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -24,7 +26,61 @@ class C
             Diagnostic(ErrorCode.ERR_FeatureInPreview, "readonly").WithArguments("ref readonly parameters").WithLocation(3, 16));
 
         CreateCompilation(source, parseOptions: TestOptions.RegularNext).VerifyDiagnostics();
-        CreateCompilation(source).VerifyDiagnostics();
+        var comp = CreateCompilation(source).VerifyDiagnostics();
+
+        var syntaxTree = comp.SyntaxTrees.Single();
+        var parameter = syntaxTree.GetRoot().DescendantNodes().OfType<ParameterSyntax>().Single();
+        Assert.Equal("ref readonly int p", parameter.ToString());
+        var model = comp.GetSemanticModel(syntaxTree, ignoreAccessibility: false);
+        var symbol = model.GetDeclaredSymbol(parameter);
```

Comment:
>var symbol = model.GetDeclaredSymbol(parameter); [](http://example.com/codeflow?start=8&length=48)

This is an unnecessary complicated way to get a parameter symbol. Pretty much every symbol declared outside of a method body can be accessed directly from compilation. We have a bunch of helpers to locate members and types, and it is one step from a method to its parameter.